datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaConnector.java

```diff
@@ -77,6 +81,8 @@
 
   private static final Duration CANCEL_TASK_TIMEOUT = Duration.ofSeconds(75);
   private static final Duration POST_CANCEL_TASK_TIMEOUT = Duration.ofSeconds(15);
+  // This should be at least equal to or greater than sum of cancel and post cancel timeouts
+  private static final Duration CANCEL_TASK_THREAD_TIMEOUT = Duration.ofSeconds(90);
```

Comment:
Why not make variables for `75` and `15`, and add them here? That'll be the best way to enforce what you're intending with your comment.